.github/workflows/test_jax_dockerfile.yml

```diff
@@ -47,9 +46,29 @@ jobs:
   test_wheels:
     name: Test | ${{ inputs.amdgpu_family }}
     runs-on: ${{ inputs.test_runs_on }}
+    strategy:
+      matrix:
+        jax_ref: [master]
     steps:
-      - name: Checkout
+      - name: Checkout JAX plugin repo
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
         with:
           repo: rocm/rocm-jax
-      # TODO: Add steps for creating the JAX docker image with an install of TheRock and then running JAX tests on the container
+          ref: ${{ matrix.jax_ref }}
+
+      - name: Checkout JAX tests repo
+        uses: actions/checkout@v4
+        with:
+          repository: rocm/jax
+          ref: ${{ inputs.jax_test_branch }}
+          path: jax
+
+      - name: Authenticate to GitHub Container Registry
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" \
+            | docker login ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
Not sure for what step this is needed but if it is required, you could rather use the docker-login action: https://github.com/marketplace/actions/docker-login#github-container-registry

Reply:
Done